test/ios-project-service.ts

```diff
@@ -121,6 +121,11 @@ function createTestInjector(projectPath: string, projectName: string, xcode?: IX
 	testInjector.register("httpClient", {});
 	testInjector.register("platformEnvironmentRequirements", {});
 	testInjector.register("plistParser", {});
+	testInjector.register("androidEmulatorServices", {});
```

Comment:
why do we register Android specific services in iOS project service?

Reply:
Because this tests relies on real `DevicesService` - https://github.com/NativeScript/nativescript-cli/blob/b1c79a471a9438a36c57e86457dcbab729812c99/test/ios-project-service.ts#L82
On the other hand `devicesService` injects `androidEmulatorServices` so we need it here.